src/actions/boards/types.js

```diff
@@ -0,0 +1,7 @@
+const types = {
+  CREATE_NEW_BOARD: 'CREATE_NEW_BOARD',
+  CREATE_BOARD_FAILED: 'CREATE_BOARD_FAILED',
+  GET_USER_BOARDS: 'GET_USER_BOARDS',
+}
```

Comment:
Try to create 3 states for every simple CRUD operation, e.x.: 

CREATE_NEW_BOARD
CREATE_NEW_BOARD_SUCCESS
CREATE_NEW_BOARD_FAILED